### PR TITLE
Update Debian Package 

### PR DESCRIPTION
**Pull request by joshughes**
It is nice to have the ability to create a debian package when your deploying statsd across your infrastructure. I found that the upstart script works but the inti.d script does not... so I removed it. 

I also wanted to have an upstart job for the proxy and install the npm dependencies when the package is installed. 

The dependencies and version were also update to be in sync with master.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -7,7 +7,7 @@ Build-Depends: debhelper (>= 8.0.0)
 
 Package: statsd
 Architecture: all
-Depends: nodejs (>= 0.6), adduser
+Depends: nodejs (>= 0.8), adduser
```
Comment (mrtazz) [inline debian/control]: can you bump this up to 0.10? We don't support 0.8 anymore

**Comment by tuxinaut** [comment]
:+1:

**Comment by mrtazz** [comment]
thanks for the contribution!

**Comment by roflmao** [comment]
So this makes debian packages not actually have a working initscript since no debian use upstart by default? sysvinit is compat with systemd at least.

**Pull request merged**